cpp/fetcher/continuous_fetcher.cc

```diff
@@ -123,23 +125,18 @@ void ContinuousFetcherImpl::FetchDone(Task* task) {
   fetch_task_.reset();
 
   if (restart_fetch_) {
-    base_->Add(bind(&ContinuousFetcherImpl::FetchDelayDone, this, nullptr));
+    executor_->Add(bind(&ContinuousFetcherImpl::FetchDelayDone, this));
   } else {
-    base_->Delay(seconds(FLAGS_delay_between_fetches_seconds),
-                 new Task(bind(&ContinuousFetcherImpl::FetchDelayDone, this,
-                               _1),
-                          base_));
+    executor_->Add([this]() {
```

Comment:
I'd rather keep ```base_``` for the delay, this kind of sleeping in a thread pool callback is the kind of thing that's getting us in trouble... :wink: